pkg/facade/computerFacade/computerFacade_test.go

```diff
@@ -0,0 +1,43 @@
+package computerFacade
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	cpuT "github.com/alextonkonogov/tonkonogov_progress/pkg/facade/cpu"
+	hardDriveT "github.com/alextonkonogov/tonkonogov_progress/pkg/facade/hardDrive"
+	memoryT "github.com/alextonkonogov/tonkonogov_progress/pkg/facade/memory"
+)
+
+var tests = []struct {
+	position string
+	lba      int
+	size     int
+}{
+	{"0x00", 62, 128},
+	{"0x2a", 63, 256},
+	{"0b10", 945, 512},
+	{"0x20", 1007, 1024},
+	{"0x3e", 1070, 2048},
+	{"0x5b", 2015, 4096},
+}
+
+func TestFacade(t *testing.T) {
+	for i, tt := range tests {
+		t.Run(strconv.Itoa(i), func(t *testing.T) {
```

Comment:
```suggestion
		t.Run(TestNameConst, func(t *testing.T) {
```
Например:
https://git.wildberries.ru/infrastructure/video-service/blob/master/pkg/service/service_test.go